packages/w3up-client/test/helpers/utils.js

```diff
@@ -1,14 +1,69 @@
-import { Receipt } from '@ucanto/core'
 import { conclude } from '@web3-storage/capabilities/ucan'
+import { parseLink } from '@ucanto/server'
+import { Receipt, Message } from '@ucanto/core'
+import * as CAR from '@ucanto/transport/car'
 import * as Server from '@ucanto/server'
 import { UCAN } from '@web3-storage/capabilities'
 import * as HTTP from '@web3-storage/capabilities/http'
 import * as W3sBlobCapabilities from '@web3-storage/capabilities/web3.storage/blob'
 import { W3sBlob } from '@web3-storage/capabilities'
 import * as Types from '../../src/types.js'
+import * as Signer from '@ucanto/principal/ed25519'
+import { Assert } from '@web3-storage/content-claims/capability'
 
 export const validateAuthorization = () => ({ ok: {} })
 
+// @ts-ignore Parameter
+export const setupGetReceipt = (contentGenFn) => {
+  // @type {Generator<import('multiformats').Link,void,unknown>} contentGenFn
+  const contentGen = contentGenFn()
```

Comment:
I'm not sure I understand why this is an async generator...why not just pass an array of links?